src/grpc/packet_router/hpr_packet_router_service.erl

```diff
@@ -38,6 +38,11 @@ route(EnvUp, StreamState) ->
                     ok = ?MODULE:register(PubKeyBin),
                     {ok, StreamState}
             end
+    catch
+        _E:_R ->
+            lager:warning("reason  ~p", [_R]),
+            lager:warning("bad envelope ~p", [EnvUp]),
+            {stop, StreamState}
```

Comment:
Should we send back some sort of response?
This way we're not killing the connection and the gateway is endlessly reconnecting and sending bad packets.
Or does that matter?

Reply:
don't think it does